projects/bucket-protocol/index.js

```diff
@@ -129,21 +155,23 @@ async function tvl(_, _1, _2, { api }) {
 
   // Cetus USDC-BUCK LP
   // 1 Bucketus = 0.5 BUCK + 0.5 USDC
+  // Bucketus doesn't include USDT, and of course not equal to USDT
```

Comment:
unsure what you mean by USDT here.

Reply:
Sorry, it's my fault. I misunderstood the purpose of the change. I'll modify it and exclude buck from tvl